compiler/core/src/swift/swiftComponent.re

```diff
@@ -451,6 +451,17 @@ let generate =
       |> List.map(defineInitialLayerValues)
       |> List.concat;
     };
+    /* isPropertyUsed should be made more general if needed in other places */
+    let isPropertyUsed = (layer: Types.layer, property) => {
+      let assignedParameters =
+        Layer.LayerMap.find_opt(layer, layerParameterAssignments);
+      let parameterIsAssigned =
+        switch assignedParameters {
+        | Some(parameters) => StringMap.mem(property, parameters)
+        | None => false
+        };
+      parameterIsAssigned || StringMap.mem(property, layer.parameters);
```

Comment:
@dabbott out of curiosity what does the `|| StringMap.mem(property, layer.parameters)` cover?

Reply:
Parameters are in two categories: some are set as the initial value for a layer, and some are assigned when executing logic (the `update` function). `StringMap.mem(property, layer.parameters)` covers the initial values

Reply:
@jasonzurita

Reply:
ahhh, good to know, thanks!